### ISSUE TITLE
StructuredListWrapper Component - Elements containing a 'region' role must be labeled with an aria-label or aria-labelledby

### ISSUE THREAD

**Issue by srlim** [carbon-design-system/carbon-components-react#338]
## Detailed description

Describe in detail the issue you're having. Is this a feature request (new component, new icon), a bug, or a general issue?

When running DAP(Dynamic Access Plugin), StructuredListWrapper gives an accessibility violation, Elements containing a 'region' role must be labeled with an aria-label or aria-labelledby, because it doesn't have aria-label.
https://aat.w3ibm.mybluemix.net/token/ea4f304d-20f3-44cb-b214-251277e22c10/d493b649-ae0e-4d33-857a-1409e4514c5c/latest///doc/w3/help/en-US/idhi_accessibility_check_g1158.html

> What browser are you working in?
Chrome 62.0.3202.94

> What version of the Carbon Design System are you using?
4.0.0

**Comment by srlim** [comment]
works now.. not sure what I was onto 😞 Closing
